src/eval.cpp

```diff
@@ -1622,10 +1622,8 @@ namespace Sass {
   // be fixed by implement superselector correctly for `:not`
   // first use of "find" (ATM only implemented for selectors)
   bool hasNotSelector(AST_Node_Obj obj) {
-    if (Wrapped_Selector_Ptr w = Cast<Wrapped_Selector>(obj)) {
-      return w->name() == ":not";
-    }
-    return false;
+    Wrapped_Selector_Ptr w = Cast<Wrapped_Selector>(obj);
+    return w && w->name() == ":not";
```

Comment:
This appears to be a style only change. If so can you please restore the previous style.

Reply:
Yes, reverted.

Reply:
Thanks